.vuepress/config.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
                     { text: 'Projects', link: '/projects/' },
                     {
                         text: 'Communities', items: [
+                            { text: 'Micro-Enterprise', link: '/micro-enterprise/' },
```

Comment:
Please put your link below the other communities.

Reply:
Position has been changed.